packages/cardano-graphql/src/StakePoolSearchProvider/query.gql

```diff
@@ -0,0 +1,75 @@
+# Review: this is not used anywhere right now,
+# but would be great to keep the schema that server uses
+# and SDK typescript schema in sync.
+# Perhaps import this npm package in server?
```

Comment:
Yes, this is the design I have in mind. We may be better off deriving the GraphQL schema from TS types though for better composition and likely less duplication. Let's chat about this.